package.json

```diff
@@ -82,6 +82,7 @@
     "eslint-plugin-react": "^7.19.0",
     "eslint-plugin-react-hooks": "^2.5.0",
     "install-peers-cli": "^2.2.0",
+    "is-cidr": "^4.0.2",
```

Comment:
Move to `dependencies`, please

Reply:
@mareklibra  same issue here with microbundle as [here](https://github.com/openshift-assisted/assisted-ui-lib/pull/455#discussion_r576960873)

Reply:
indeed :-)